pkg/apis/serving/v1alpha2/inferenceservice_types.go

```diff
@@ -88,6 +91,20 @@ type Logger struct {
 	Mode LoggerMode `json:"mode,omitempty"`
 }
 
+// Batcher provides optional payload batcher for all endpoints
+// +experimental
+type Batcher struct {
+	// MaxBatchSize of batcher service
+	// +optional
+	MaxBatchSize *string `json:"maxBatchSize,omitempty"`
+	// MaxLatency of batcher service
+	// +optional
+	MaxLatency *string `json:"maxLatency,omitempty"`
```

Comment:
Just realized here we need validations for the string? should we use the actual int/float type here?

Reply:
All the values are string type.